Testnet/StandardTokenOwnableDynamicSupply/StandardTokenOwnableDynamicSupply/StandardTokenOwnableDynamicSupplyContract.cs

```diff
@@ -192,6 +188,12 @@ public void Mint(Address account, UInt256 amount)
         Log(new TransferLog() { From = Address.Zero, To = account, Amount = amount });
 
         this.TotalSupply += amount;
+
+        Log(new SupplyChangeLog()
+        {
+            PreviousSupply = (this.TotalSupply - amount),
+            TotalSupply = this.TotalSupply
+        });
```

Comment:
Very minor but you could save some gas here with the following, same sort of comment for burn. 

``` c#
var totalSupply = this.TotalSupply;
var updatedSupply = totalSupply += amount;

TotalSupply = updatedSupply;

Log(new SupplyChangeLog()
{
    PreviousSupply = totalSupply,
    TotalSupply = updatedSupply
});